code/modules/mob/living/life.dm

```diff
@@ -121,7 +121,7 @@
 	return TRUE
 
 /mob/living/proc/experiences_hunger_and_thirst()
-	return TRUE
+	return !isSynthetic() // Doesn't really apply to robots. Maybe unify this with cells in the future.
```

Comment:
really wary of an isSynthetic check, this should probably be a bodytype check if we're doing it right. i get wanting to slap in a quick fix though